install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -111,6 +111,69 @@ spec:
 labels of targeted PV.
 ====
 
+[[selector-label-volume-bind]]
+=== Optional: Bind a PVC to a specific PV
+
+A PVC that does not specify a PV name or selector will match any PV.
+
+To bind a PVC to a specific PV as a cluster administrator:
+
+* Use `pvc.spec.volumeName` if you know the PV name.
+* Use `pvc.spec.selector` if you know the PV labels.
++
+By specifying a selector, the PVC requires the PV to have specific labels.
+
+
+[[selector-label-volume-reserve]]
+=== Optional: Reserve a PV to a specific PVC
+
+To reserve a PV for specific tasks, you have two options: create a specific storage class, or pre-bind the PV to your PVC.
+
+. Request a specific storage class for the PV by specifying the storage class’s name.
++
+The following resource shows the required values that you use to configure a StorageClass. This example uses the AWS ElasticBlockStore (EBS) object definition.
+
++
+.StorageClass definition for EBS
+====
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: kafka
+provisioner: kubernetes.io/aws-ebs
+...
+----
+====
++
+[IMPORTANT]
+====
+If necessary in a multi-tenant environment, use a quota definition to reserve the storage class and PV(s) only to a specific namespace.
+====
+
+. Pre-bind the PV to your PVC using the PVC namespace and name. A PV defined as such will bind only to the specified PVC and to nothing else, as shown in the following example:
```

Comment:
@chao007 - I dropped "namespace" from the subhead as you suggested, but is it okay to still have it "PVC namespace" here or should it also be removed?